src/rv2/samples/workflow-configs/object-detection/cowc-potsdam-test.json

```diff
@@ -1,33 +1,34 @@
 {
     "train_projects": [
         {
+            "id": "2-10",
             "raster_source": {
                 "geotiff_files": {
                     "uris": [
-                        "{raw}/isprs-potsdam/4_Ortho_RGBIR/top_potsdam_2_10_RGBIR.tif"
+                        "{rv_root}/processed-data/cowc-potsdam-test/2-10.tif"
                     ]
                 }
             },
             "ground_truth_label_source": {
                 "geojson_file": {
-                    "uri": "{rv_root}/processed-data/cowc-potsdam/labels/train/top_potsdam_2_10_RGBIR.json"
+                    "uri": "{rv_root}/processed-data/cowc-potsdam-test/2-10.json"
                 }
             }
         }
     ],
     "test_projects": [
         {
-            "id": "2_13",
+            "id": "2-13",
             "raster_source": {
                 "geotiff_files": {
                     "uris": [
-                        "{raw}/isprs-potsdam/4_Ortho_RGBIR/top_potsdam_2_13_RGBIR.tif"
+                        "{rv_root}/processed-data/cowc-potsdam-test/2-13.tif"
```

Comment:
Why move raw to rv_root? The separation was useful in my run against the s3 images without copying

Reply:
This is just for the `-test.json` workflow. This workflow uses a tiny dataset (containing the imagery and labels) in order to run faster.

Reply:
Ahh gotcha, understood.